--- /Users/john/iwish/src/androwish/jni/tkpath/generic/path.c	2026-06-11 10:26:32
+++ /Users/john/undroidwish91/ext-build/tkpath-src/generic/path.c	2026-07-19 18:34:14
@@ -18,7 +18,7 @@
 /* Keep patch level release numbers odd and set even only on release. */
 #define TKPATH_VERSION    "0.3"
 #define TKPATH_PATCHLEVEL "0.3.3"
-#define TKPATH_REQUIRE    "8.5"
+#define TKPATH_REQUIRE    "8.5-"
 
 MODULE_SCOPE int gAntiAlias;
 MODULE_SCOPE int gSurfaceCopyPremultiplyAlpha;
